src/ApiPort.VisualStudio.Common/ApiPort.VisualStudio.Common.csproj

```diff
@@ -47,6 +47,7 @@
     <ProjectReference Include="..\Microsoft.Fx.Portability\Microsoft.Fx.Portability.csproj">
       <Project>{8d84ec23-9977-4cc8-b649-035ffae9664c}</Project>
       <Name>Microsoft.Fx.Portability</Name>
+      <AdditionalProperties>TargetFramework=net46</AdditionalProperties>
```

Comment:
This is going to be a continual pain. The initial problem is fixed (see my other comment and #482)